docker-bake.hcl

```diff
@@ -332,6 +331,10 @@ target "cloud-tools-latest" {
         inherits = [
                 "virtual-cloud-tools",
         ]
+        platforms = [
+                "linux/amd64",
+                "linux/arm64",
+        ]
```

Comment:
This should be in the virtual-cloud-tools target, since we want this for other base versions as well, right? There are no other base targets, yet, but they might appear any moment when a fedora upgrade breaks our CI.

Reply:
Updated.